rust/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -736,6 +733,14 @@ impl Indexer<H256> for SealevelMailboxIndexer {
     }
 }
 
+#[async_trait]
+impl SequenceIndexer<H256> for SealevelMailboxIndexer {
```

Comment:
This is for indexing deliveries right? Worth changing the log msg on 739, and maybe a comment about us implementing this when we build scraper support for sealevel